lib/svg-sprite/fix-xml-string.js

```diff
@@ -0,0 +1,22 @@
+const { DOMParser } = require('@xmldom/xmldom');
+
+module.exports = svgString => {
+    let domParserError = false;
+    const errorHandler = () => {
+        domParserError = true;
+    };
+
+    const fixedSVG = new DOMParser({ errorHandler })
+        .parseFromString(svgString)
+        .toString()
+        .replace(/(\s)(\s+)/g, ' ');
+
+    if (!domParserError) {
+        return fixedSVG;
+    }
+
+    const error = new Error('Invalid XML string');
+    error.name = 'XMLFixingError';
```

Comment:
Does it still make sense to keep using custom Error Objects? I mean, we don't test them and it's probably enough if we have a clear error message only anyway.